base/process.h

```diff
@@ -16,7 +17,9 @@ namespace base {
 
   pid get_current_process_id();
 
-  bool is_process_running(pid pid);
+  std::string get_process_name(pid pid);
```

Comment:
I think you should include <string> to avoid the errors reported by Github actions.